src/main/java/racingCar/model/Cars.java

```diff
@@ -0,0 +1,39 @@
+package racingCar.model;
+
+import racingCar.utils.RandomUtil;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+    private List<Car> carList;
```

Comment:
```suggestion
    private List<Car> cars;
```

변수명에 자료형이 있을 경우 어떠한 문제가 있을지 고민해보아요 :)

Reply:
해당 변수 자료형이 바뀔때 list라는 정보가 담긴 이름도 바껴야 하기 때문이군요 :3